backend/data/processors/manualFixes/COMMFixes.py

```diff
@@ -0,0 +1,333 @@
+        "processed": "must be in Good Academic Standing"
+    """
+    return ""
+
+def COMM_1100(conditions):
+    """
+        "original": "Students who have completed ECON1101 are not permitted to enrol. <br/><br/>",
+
+        "processed": "who have ECON1101 are not permitted to enrol"
+    """
+    return {
+        "original": conditions["original"],
+        "processed": "",
+        "warning": "Students enrolled in Actuarial Studies or Economics programs (in single or double degree mode) are not permitted to enrol."
+    }
+
+def COMM_1110(conditions):
+    """
+        "original": "Only available to single and double degree Business School students in Term 1. It will be offered to non-Business School students in Terms 2 and 3.<br/><br/>",
+        "processed": "Only available to single && double degree Business School in Term 1. It will be offered to non-Business School in Terms 2 && 3"
+    """
+
+    return {
+        "original": conditions["original"],
+        "processed": "",
+        "warning": "Students enrolled in 3764 (Eng/Comm), Actuarial Studies or Economics programs (in both single and double degree mode) are not permitted to enrol."
+    }
+
+# def COMM_1120(conditions):
+    """
+        "original": "Only available to single and double degree Business School students in Term 1. It will be offered to non-Business School students in Terms 2 and 3.<br/><br/>",
+        "processed": "Only available to single && double degree Business School in Term 1. It will be offered to non-Business School in Terms 2 && 3"
+    """
+
+    return {
+        "original": conditions["original"],
+        "processed": conditions["processed"],
+        "warning": "Only available to single and double degree Business School in Term 1. It will be offered to non-Business School in Terms 2 and 3."
+    }
+
+# def COMM_1140(conditions):
+    """
+        "original": "Only available to single and double degree Business School students in Term 1. It will be offered to non-Business School students in Terms 2 and 3.<br/><br/>",
+        "processed": "Only available to single && double degree Business School in Term 1. It will be offered to non-Business School in Terms 2 && 3"
+    """
+
+    return {
+        "original": conditions["original"],
+        "processed": conditions["processed"],
+        "warning": "Only available to single and double degree Business School in Term 1. It will be offered to non-Business School in Terms 2 and 3."
+    }
+
+def COMM_1150(conditions):
+    """
+        "original": "Pre-requisite: COMM1100 and excludes MGMT1101. Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Term 3.<br/><br/>",
+        "processed": "COMM1100 &&"
+    """
+
+    COURSES["COMM1150"]["exclusions"]["MGMT1101"] = 1
+    return {
+        "original": conditions["original"],
+        "processed": "COMM1100",
+    }
+
+# def COMM_1170(conditions):
```

Comment:
Likewise here and below, remember to comment out the entire function or else there's a bunch of floating returns (which probably breaks the code when we run it)